insighthub-backend/src/config.ts

```diff
@@ -0,0 +1,14 @@
+export const config = {
+    mongo: {
+        uri: process.env.DB_CONNECTION || 'mongodb://localhost:27017'
+    },
+    app: {
+        port: process.env.PORT || 3000,
+        client_url: process.env.CLIENT_URL || 'http://localhost:5000'
```

Comment:
1. Isn't this supposed to be an arrow function too?
2. Can you please document this new environment variable in `insighthub-backend/README.md` and `insighthub-backend/env.template` ?

Reply:
Fixed in cde95a5de3f9d1b67dc637060259fb71507ba7af